examples/evaluation/README.md

```diff
@@ -9,6 +9,10 @@ specified dataset using builtin default evaluator, and log resulting metrics & a
   generated by `sklearn.datasets.make_classification`.
 - Example `evaluate_on_regressor.py` evaluate as scikit-learn `LinearRegression` model on dataset loaded by
   `sklearn.datasets.fetch_california_housing`
+- Example `evaluate_with_custom_metrics.py` evaluates a custom metric function on a scikit-learn `LinearRegression`
```

Comment:
"Evaluate a custom metric function" isn't quite correct terminology. Models are evaluated on datasets, producing a collection of metrics, which may be computed using custom routines. Can we rephrase?